mlflow/sklearn/__init__.py

```diff
@@ -1182,6 +1183,12 @@ def fetch_logged_data(run_id):
     :param registered_model_name: If given, each time a model is trained, it is registered as a
                                   new model version of the registered model with this name.
                                   The registered model is created if it does not already exist.
+    :param pos_label: If given, used as the positive label to compute binary classification
+                      training metrics such as precision, recall, f1, etc. This parameter
+                      should only be set for binary classification model
+                      - if used on multi-label model, the training metrics calculation will fail
+                        and the training metrics won't be logged.
+                      - if used for regression model, the parameter will be ignored.
```

Comment:
@wenfeiy-db Can you fix this doc build error: https://app.circleci.com/pipelines/github/mlflow/mlflow/12552/workflows/cb9ac64c-2368-4844-907e-37547ffef633/jobs/21422?invite=true#step-104-62?

Reply:
The doc build now passes and the page LGTM.
![image](https://user-images.githubusercontent.com/87323464/175698629-19e7939a-7693-4c4a-ba76-34ac28f95dd4.png)